test/test.js

```diff
@@ -721,7 +721,7 @@ test("Mix", function () {
 
```

Comment:
Can you also add an 'equal' case for two colors that was failing before your change and now is passing?  After the check on line 715 would be a good place for that